src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+        class="formBig"
```

Comment:
You have similar styles for both forms. So, use a combination of the class names
```suggestion
        class="search-form search-form-big"
```